posix.mak

```diff
@@ -110,7 +112,7 @@ test_tests_extractor: $(ROOT)/tests_extractor
 	$< -i ./test/tests_extractor/iteration.d | diff - ./test/tests_extractor/iteration.d.ext
 
 test_rdmd: $(ROOT)/rdmd_test $(ROOT)/rdmd
-	$< --compiler=$(abspath $(DMD)) -m$(MODEL)
+	$< --compiler=$(abspath $(DMD)) -m$(MODEL) -v --test-compilers=$(RDMD_TEST_COMPILERS)
```

Comment:
On consideration, I think it could be useful to keep the verbose call to `rdmd_test` for CI at least.  But it's a bit intrusive how it's done here, just hardcoding the `-v`.  Any thoughts from anyone else?

If we decide to keep it as-is, I should probably tweak the last commit message.

Reply:
Are you looking for something like this:

```make
VERBOSE=0
...
ifeq ($(VERBOSE),1)
    override VERBOSE_FLAGS:=-v
fi
...
$< ... $(VERBOSE_FLAGS)
```

in Travis you can set `VERBOSE=1`

> But it's a bit intrusive how it's done here

I don't think anyone would care. Testsuites are supposed to be chatty and emit a lot of messages.

Reply:
Yes, I can do something like that.  I was not going to because I'd assumed the last patch would just be something to drop after things had been finalized.  But given that the output seems useful, I think I'll revise things accordingly.